charts/operator/templates/deployment.yaml

```diff
@@ -47,6 +47,8 @@ spec:
         - --health-probe-bind-address=:8081
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
+        - --configurator-tag={{ include "operator.tag" . }}
+        - --configurator-base-image=redpandadata/configurator
```

Comment:
```suggestion
        - --configurator-base-image=docker.redpanda.com/redpandadata/configurator
```

Reply:
Even better I replaced the hardcoded values with the values.yaml.